hphp/runtime/base/runtime-option.h

```diff
@@ -606,6 +606,12 @@ struct RuntimeOption {
   F(int64_t, PerfWarningSampleRate, 1)                                  \
   /* Profiling flags */                                                 \
   F(bool, EnableReverseDataMap, false)                                  \
+  /* PPC64 Option: Minimum immediate size to use TOC */                 \
+  F(uint16_t, PPC64minTOCImmSize, 64)                                   \
+  /* PPC64 Option: Dump the number of elements stored in TOC */         \
+  F(bool, PPC64dumpTOCNelements, false)                                 \
+  /* PPC64 Option: Allow lwz and not only ld to be used for TOC read */ \
+  F(bool, PPC64useTOCLwz, false)                                        \
```

Comment:
Capitalize the `u` in `use`, and the `m` in `min` above.

Why do these need to be configurable values?  Can you explain when and why you would ever want to change them?